Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionCiscoISE.yaml

```diff
@@ -0,0 +1,81 @@
+Parser:
```

Comment:
Please update union ASIM and VIM parser files by adding this parser.

Reply:
updated